src/ResourceManager/Network/Commands.Network/help/New-AzureRmApplicationGateway.md

```diff
@@ -394,6 +399,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -RewriteRuleSets
```

Comment:
@sai-manoj-kumar the help for this will need to be regenerated when the parameter is changed to `RewriteRuleSet`

Reply:
Same as the code change. To keep in sync with other parameters for this cmdlet, not changing this parameter.